bin/cli.js

```diff
@@ -193,30 +217,37 @@ const getConfig = () => {
 
   config.files = merge(config.files, program.args);
   config.files = loadFiles(config.files);
-
   config.exclude = merge(config.exclude, program.exclude);
   config.files = exclude(config.files, config.exclude);
   config.nodeOnly = merge(config.nodeOnly, program.nodeOnly);
   config.browserOnly = merge(config.browserOnly, program.browserOnly);
+  config.logLevel = program.logLevel || 'default';
+  config.reporter = program.logLevel || program.reporter || 'default';
 
   if (config.environments.includes('browser')) {
     config.browser = config.browser || {};
-
-    config.browser.browsers = merge(
-      config.browser.browsers, program.browsers);
-    config.browser.logLevel = program.browserLog || config.browser.logLevel;
+    config.browser.browsers = merge(config.browser.browsers, program.browsers);
+    config.browser.logLevel = program.browserLog ||
+      program.logLevel ?
+      convertLogLevel(program.logLevel) :
```

Comment:
Don't we need indentation in ternary operator?

Reply:
Apparently, current eslint config requires indentation of 6 spaces. In order to do so it should be wrapped in parentheses, than this should be fine.